### ISSUE TITLE
Node sdk leading to large memory/cpu utilization

### ISSUE THREAD

**Issue by csydvs** [featurehub-io/featurehub#488]
**Describe the bug**
Node sdk seems to be maintaining a context for every call and never letting them go, leading to large cpu/memory utilization on the node server and out of memory errors on the edge service side.

 Which area does this issue belong to?

* [ ] FeatureHub Admin Web app
* [X] SDK
* [ ] SDK examples
* [ ] Documentation
* [ ] Other

**To Reproduce**
I'm running a graphql node server using featurehub-javascript-node-sdk:1.0.5. Right at startup I initiailze an EdgeFeatureHubConfig using something like this:
```
      featurehubConfig = new EdgeFeatureHubConfig(
        featureHubUrlL,
        featureHubSecret
      ).init()
```
Then, when I want to check a feature, I create a new context for that user, with something like this:
```
let ctx = featurehubConfig.newContext()
// Add ctx.userKey() and ctx.attribute_value()
await ctx.build()
```
Once my context is created, I call `ctx.isEnabled(featureString)`. 

What I'm seeing in the logs of my local party server is traffic a minute after I create the config (in perpetuity), but also I see traffic from every ctx that's been created every minute on the minute as well. I've tried calling `ctx.close()`, and that hasn't improved anything. This taken altogether ends up making my graphql server take a ton of memory and cpu trying to maintain all of these contexts, and also makes my edge servers start getting out of memory errors in production as well, all for contexts that I'm never going to use again.

**Expected behavior**
I think it makes sense that the config would keep some sort of connection open to the server, but I need a way to be done with contexts when I'm done with them, and not have the additional cost of maintaining them forever.

**Versions**
I'm keeping track of connections using the logs on a party server docker implementation running 1.3.3. However, I'm seeing the expensive memory consumption on an all separate stack, also running 1.3.3.
I'm using `featurehub-javascript-node-sdk:1.0.5`

**Comment by rvowles** [comment]
Hi there! Thanks for this ticket, could you tell me one more thing? Which type of key are you using, a client side eval or a server side eval?

Thanks!

**Comment by csydvs** [comment]
Hey Richard, thanks for reaching out. I'm using a client side eval key in this particular server, although I've made use of both in my system depending on whether it's a server I'm running or in the browser.

**Comment by rvowles** [comment]
@csydvs there is a PR up now that resolves this issue

**Comment by rvowles** [comment]
Sorry for the unceremonious closing - Github just does that when a fix merges. Please re-open if it doesn't do it for you :-)

**Comment by IrinaSouth** [comment]
This fix has been released in version 1.0.6 for [Node](https://www.npmjs.com/package/featurehub-javascript-node-sdk) and [Client SDK ](https://www.npmjs.com/package/featurehub-javascript-client-sdk)
